.github/workflows/upload-release-artifact.yml

```diff
@@ -5,7 +5,7 @@ on:
     types: [ published ]
 
 env:
-  PROJECT: minio-plugin
+  ARTIFACT_ID: minio
```

Comment:
I confused myself with this env name yesterday, so i fixed it and renamed it to what it really describes